boilerplate/app/screens/DemoPodcastListScreen.tsx

```diff
@@ -77,28 +91,126 @@ const EpisodeCard = observer(function EpisodeCard({
+   * @see https://reactnative.dev/docs/accessibility#accessibilityactions
+   */
+  const accessibilityHintProps = useMemo(
+    () =>
+      Platform.select<AccessibilityProps>({
+        ios: {
+          accessibilityHint: translate("demoPodcastListScreen.accessibility.cardHint", {
+            action: isFavorite ? "unfavorite" : "favorite",
+          }),
+        },
+        android: {
+          accessibilityLabel: episode.title,
+          accessibilityActions: [
+            {
+              name: "longpress",
+              label: translate("demoPodcastListScreen.accessibility.favoriteAction"),
+            },
+          ],
+          onAccessibilityAction: ({ nativeEvent }) => {
+            if (nativeEvent.actionName === "longpress") {
+              handlePressFavorite()
+            }
+          },
+        },
+      }),
+    [episode, isFavorite],
+  )
+
+  const handlePressFavorite = () => {
+    onPressFavorite()
+    liked.value = withSpring(liked.value ? 0 : 1)
+  }
+
+  const handlePressCard = () => {
+    openLinkInBrowser(episode.enclosure.link)
+  }
+
   return (
+    //  MAVERICKTODO: Switch this out for Card component when that is ready
     <TouchableOpacity
       style={[$rowLayout, $item]}
-      onPress={() => openLinkInBrowser(episode.enclosure.link)}
+      onPress={handlePressCard}
+      onLongPress={handlePressFavorite}
+      // MAVERICKTODO: This button role should be set on the Card Component
+      accessibilityRole="button"
+      {...accessibilityHintProps}
     >
       <View style={$description}>
         <Text>{episode.title}</Text>
         <View style={[$rowLayout, $metadata]}>
-          <Icon
-            icon="heart"
-            color={isFavorite ? colors.palette.primary400 : undefined}
-            onPress={onPressFavorite}
-          />
-          <Text size="xs">{episode.datePublished}</Text>
-          <Text size="xs">{episode.duration}</Text>
+          <Animated.View
+            style={[$iconContainer, StyleSheet.absoluteFillObject, animatedLikeButtonStyles]}
+          >
+            <Icon
+              icon="heart"
+              size={ICON_SIZE}
+              color={colors.palette.neutral800} // dark grey
+              onPress={handlePressFavorite}
+              accessibilityLabel={
+                isFavorite
+                  ? undefined
+                  : translate("demoPodcastListScreen.accessibility.favoriteIcon")
+              }
+            />
+          </Animated.View>
+          <Animated.View style={[$iconContainer, animatedUnlikeButtonStyles]}>
+            <Icon
+              icon="heart"
+              size={ICON_SIZE}
+              color={colors.palette.primary400} // pink
+              onPress={handlePressFavorite}
+              accessibilityLabel={
+                isFavorite
+                  ? translate("demoPodcastListScreen.accessibility.unfavoriteIcon")
+                  : undefined
```

Comment:
undefined if not `isFavorite`? why can't we just always use the one translation here, since they're different components when favorite/not favorite?

Reply:
The issue is they both exist on screen due to the way the animation works, one just gets set to 0 opacity. So I don't want it to read out the one that's not active.